test/models/organization_test.rb

```diff
@@ -1,6 +1,64 @@
 require 'test_helper'
 require 'models/shared/taxonomies_base_test'
+require 'rfauxfactory'
 
 class OrganizationTest < ActiveSupport::TestCase
   include TaxonomiesBaseTest
+
+  # List of valid organization names.
+  def valid_org_name_list
+    # Note: The maximum allowed length of org name is 242 only. This is an
+    # intended behavior (Also note that 255 is the standard across other
+    # entities.)
+    [
+      RFauxFactory.gen_alpha(1),
+      RFauxFactory.gen_alpha(242),
+      *RFauxFactory.gen_strings(1..242, exclude: [:html]).values,
+      RFauxFactory.gen_html(rand((1..217)))
+    ]
+  end
+
+  # List of invalid organization names.
+  def invalid_org_name_list
+    [
+      '',
+      ' ',
+      "\t",
+      *RFauxFactory.gen_strings(243).values
+    ]
+  end
+
+  test "create with multi names" do
+    valid_org_name_list.each do |name|
+      organization = FactoryBot.create(:organization, :name => name)
```

Comment:
I think we do not need to save the model which, SQL queries are slow and we only want to ensure that validation catches it. If you write the test like tihs, it will have the same impact but will be faster since it only builds in-memory objects and performs validation only. Also it could be more fragile if the save would start failing because of other reasons than name validity.

```ruby
organization = FactoryBot.build(:organizaton, :name => name)
assert organization.valid?
```

more technical details (but I guess you already know since I saw the test with invalid name):
In rails, when you call `save` it calls `valid?` inside and does not trigger the SQL query if it returns false. `valid?` runs all models validations and returns false if some of these registered some error on this model. You could argue that your test is more thorough since it also verifies behavior of `save`, but we should rely on tests of rails framework that covers it already.